devicemodel/core/main.c

```diff
@@ -999,10 +998,10 @@ main(int argc, char *argv[])
 					errx(EX_USAGE, "invalid pm_notify_channel: %s", optarg);
 			} else
 				errx(EX_USAGE, "invalid pm_notify_channel: %s", optarg);
+			pr_warn("The \"--pm_notify_channel\" parameter is obsoleted\n");
```

Comment:
It probably does not make sense to check whether the `pm_notify_channel` parameters are valid (just immediately above), does it?

Reply:
Yes, we need keep the interfaces for now. 
But We will print a log to indicate the parameters are obsoleted but not ignore.